### ISSUE TITLE
Links to targets in collapsed sections should open them

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetTools#1235]
In DoenetML, if one makes a <ref> to a target that is in a section that is closed, clicking on the link should open the section and the page should scroll to the location of the target.  Similar behavior should occur if one enters a URL with the anchor corresponding to the target.

```
<p>See <ref target="note1">note 1</ref>.</p>
<aside>
  <title>Notes</title>
  <ol>
    <li name="note1">Important note</li>
    <li name="note2">Other note</li>
  </ol>
</aside>
```

**Comment by dqnykamp** [comment]
Resolved by #1480

**Comment by dqnykamp** [comment]
Reopening, because #1480 only opens collapsed sections if link to the section itself.  Linking to targets inside collapsed sections still do not open the sections.
